### DISCUSSION TITLE
Release @ark-ui/react@1.2.2 with new Zag version

### DISCUSSION THREAD

**Discussion by juliomuhlbauer** [chakra-ui/ark#1985] [General]
For now, I can`t use Ark because of this issue: https://github.com/chakra-ui/zag/discussions/1088
It was fixed in Zag in 0.32. If we can release a new version to solve this issue, I would appreciate a lot!

**Comment by Omikorin** [comment]
I need to ping ~~manager~~ @cschroeter ;D

Anyway, @juliomuhlbauer how it was fixed? I haven't checked that yet

**Reply by juliomuhlbauer** [reply]
This is the PR: https://github.com/chakra-ui/zag/pull/1095
It was some problem with the zag store

**Comment by cschroeter** [comment]
@Omikorin 

We can do a release. Can you confirm that:

```mdx
### Added

- Added the `Progress` component.

### Fixed

- Added missing `placeholder` prop to `SelectValueTextProps` type.
- Changed `PopoverDescriptionProps` type from `div` to `p`
```

this is correct? We may wanto add the changes from Zag.js as well if we updated a veresion

**Reply by Omikorin** [reply]
I'll check later today. We should add info about the segmented control and tabs styling method update for easier migration.
